syntax = "proto3";

package demo;

message RequestData {
    string data = 1;
}

message ResponseData {
    int64 return_code = 1;
    string message = 2;
    string data = 3;
}

service DemoService {

    rpc ApiCreate(RequestData) returns (ResponseData) {}
    rpc ApiUpdate(RequestData) returns (ResponseData) {}
    rpc ApiList(RequestData) returns (ResponseData) {}
    rpc ApiDelete(RequestData) returns (ResponseData) {}

}
